### PR TITLE
Please review our code

### PR DESCRIPTION
**Pull request by Anna-Myzukina**

### CODE REVIEW

**Comment by mcrd25** [comment]
# Currently under review by @mcrd25

File: .stickler.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+linters:
+  stylelint:
```
Comment (mcrd25) [inline .stickler.yml]: You should not be using stylelint for this project. This is a ruby project. Your linter should be rubo cop as seen in the [linter repo](https://github.com/microverseinc/linters-config/tree/master/ruby)
It should look like following:

```yml
linters:
  rubocop:
    display_cop_names: true
files:
  ignore:
    - "bin/*"
    - "db/*"
    - "config/*"
    - "Guardfile"
    - "Rakefile"
```

File: bubble-sort.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    for i in 1..n
+        for j in 0..n-1
+            arr[j], arr[j+1] = arr[j+1], arr[j] if arr[j] > arr[j+1]
+        end
+    end
+    arr
+end
+
+def bubble_sort_by(arr)
+    n = arr.size - 1
```
Comment (mcrd25) [inline bubble-sort.rb]: Should use 2 not 4 space indentation

**Review by mcrd25 — CHANGES_REQUESTED** [review]
# Great job! Now for the linting! 💪 

Hi @Anna-Myzukina and @macnick 👋,

First of all team, this implementation looks great. Your README looks great. I'd give you the 👍  right away however there is one concern we have to address and another for us to consider for next Ruby project.

### Linters
- Address `.stickler.yml` review comment
- You are missing the `.rubocop.yml` as seen in the  [linter](https://github.com/microverseinc/linters-config/tree/master/ruby) repo.
- Follow all instructions in the README of the link above to activate your linter

### GitHub Flow
**You are supposed to be using GitHub Flow for this project.** Please, you must use in next Ruby project (until Tic Tac Toe) that you are to use GitHub flow. Which is basically a pull request from the feature branch to master branch. You won't be needing a develop branch. Read more about GitHub Flow [here](https://guides.github.com/introduction/flow/). The idea of GitHub flow is to ensure that the master branch is always deployable. 
I noticed you have a feature branch named `bubble_sort_method` but I see no PR associated with it. Either way, it's too much work to change now. But please remember to use GitHub Flow untill it's time to use GitFlow. Thanks 🙏 

### Things to consider for the next project
- Please make sure to use GitHub Flow in the next Ruby project.
- Try to put a brief summary description in your pull request of all the changes you've made.
For example, I have an app that I set up and I added user model, comment model, post models, and related associations and migrations. I would mention this in my pull request description
```markdown
# Summary of changes made
- Generated User, Comment and Post Models
- Created Associations between User, Comment and Post Models
- Whatever other change you'd have made
```

**Other than the issues mentioned above your project is 💯 and passes all tests.**

*Tip before adding linter 💡: Your indentation for your ruby code is not 2 spaces. This would give a linting error. Please make them 4 spaces*

***Please makes changes and submit another review request.
If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email.***

*Happy coding 😄,*
**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/) [GitLab](https://gitlab.com/mcrd25)

---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

File: .stickler.yml (hunk 1 above)
Comment (Anna-Myzukina) [reply .stickler.yml]: Thank`s for your review! You give good advice about rubo cop.
About git-flow I`m confused cause here https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/57421588/  in the dashboard we have instructions to this task and in part "Remember to:" you can see these two lists: 
-Follow the Github Flow guidelines.
Do not work directly in the master branch. 
-Create a feature-branch branch (give it a descriptive name) for the feature you implement (we assume that this entire project is one feature for learning purposes.) and open a Pull Request from it. When your feature is ready and Pull request approved, you can merge it to the master branch.

**Review by bolah2009 — CHANGES_REQUESTED** [review]
# Awesome Job Team :clap: 
## Good job implementing changes requested by @mcrd25 :blush: 
You are almost done :smile: 
- Let's Fix the linter errors/warnings (Currently, there're `11 offenses detected` locally)
   - Have you [installed stickler](https://github.com/apps/stickler-ci) and [enabled it](https://stickler-ci.com/) in this repo? [See here](https://github.com/microverseinc/linters-config/tree/master/ruby) for detailed instruction.
    - With `rubocop -a` you can fix almost all of the errors, if you have the `rubocop` gem installed: (`gem install rubocop`). :wink: 
- Update PR Description
   update-PR-description

#### :recycle:  Make these changes and submit another code review request. :+1: 
#### Feel free to reach me on slack if you having issues implementing these features. :+1: 

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

**Comment by macnick** [comment]
Thank you for your review. We corrected all the errors using rubocop -a. 

Please review our code again.

**Review by bolah2009 — APPROVED** [review]
## It's a :+1: from me team :smile: 

#### I strongly recommend updating this PR description.
   update-PR-description

## Go ahead and merge to master :rocket: 

#### Also, don't forget to submit project completion form. :blush: 

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

File: .stickler.yml (hunk 1 above)
Comment (mcrd25) [reply .stickler.yml]: Yes, so the feature branch should be appropriately named. `develop` branch is used in Git Flow. So your branch could have been named `bubble_sort` etc. and that would be the branch you do the PR on. But this is fine. Just ensure that your feature branch is appropriately named. It is probably best to not name any of your feature branches in future `developer` or `dev` or `development` or similar as a development branch is used in a different way in Git Flow. In GitHub flow only master and feature branches. Does this make sense? Hope it helps 😅 🙂

**Pull request merged**